app/selenium_ui/conftest.py

```diff
@@ -1,6 +1,7 @@
 import atexit
 import csv
 import datetime
+import glob
```

Comment:
why do you need this package?

Reply:
I thought it was before.
I will remove it